src/components/Accordion/Accordion.tsx

```diff
@@ -61,20 +62,26 @@ export const Accordion = (props: AccordionProps): React.ReactElement => {
     'usa-accordion',
     {
       'usa-accordion--bordered': bordered,
+      'aria-multiselectable': multiselectable,
```

Comment:
I don't think this is supposed to be a CSS class! instead, the Accordion `<div>` should have the `aria-multiselectable` attribute if multiselectable is true (I'm comparing with the source code at https://designsystem.digital.gov/components/accordion/)

Reply:
Fixed, thank you for catching that.